File: module-player/src/main/java/com/zhf/player/IVideoPlayerView.java
package com.zhf.player;

import java.util.ArrayList;

import com.zhf.base.activity.IBaseView;
import com.zhf.common.contract.BaseCustomViewModel;

/**
 * 应用模块:
 * <p>
 * 类描述:
 * <p>
 */
public interface IVideoPlayerView extends IBaseView {
    /**
     * 数据加载完成
     *
     * @param viewModels data
     */
    void onDataLoadFinish(ArrayList<BaseCustomViewModel> viewModels);
}

File: module-player/src/main/java/com/zhf/player/IVideoPlayerView.kt
package com.zhf.player

import com.zhf.base.activity.IBaseView
import com.zhf.common.contract.BaseCustomViewModel
import java.util.*

/**
 * created by demoless on 2021/1/29
 * description:
 */
interface IVideoPlayerView: IBaseView {
    /**
     * 数据加载完成
     *
     * @param viewModels data
     */
    fun onDataLoadFinish(viewModels: ArrayList<BaseCustomViewModel?>?)
}